src/Form/ContactMessageForm.php

```diff
@@ -36,8 +37,7 @@ public function form(array $form, FormStateInterface $form_state): array {
     // Checkbox to accept privacy policy configured in the ContactForm.
     $form['privacy_policy'] = [
       '#type' => 'checkbox',
-      '#title' => $this->t('I accept privacy policy'),
-      '#description' => $contact_form->getThirdPartySetting('oe_contact_forms', 'privacy_policy'),
+      '#title' => $this->t('Check to accept our <a href="@link" target="_blank">privacy policy</a>', ['@link' => $privacy_link]),
```

Comment:
This shoud have "I have read and agree with the <a href="@link">data protection terms</a>".